.gitignore

```diff
@@ -0,0 +1 @@
+build/
```

Comment:
This doesn't need to be here, really, as the name of the build directory is user-controlled, it could be anything (even if that one matches the instructions in the README).

Reply:
Side note: recent versions of Meson will automatically generate a `.gitignore` in the build directory.